default.lcf

```diff
@@ -97,6 +97,10 @@ log4j.category.com.strangeberry=OFF
 log4j.category.javax.jmdns=OFF
 
 # Examples of changing priority of specific categories (classes, packages):
+ log4j.category.jmri.jmrit.logix.Engineer=DEBUG
```

Comment:
Can you please revert the changes to default.lcf? We don't want the warrants logging at DEBUG level by default.